packages/injective-std/src/types/injective/types/v1beta1.rs

```diff
@@ -0,0 +1,12 @@
+// @generated
```

Comment:
Hmm, I think this would better be generated from the original proto file, along with the rest.

Reply:
Doesn't look like there are any instructions at packages/injective-std/README.md to generate all of the code at once. Only instructions for individual files. Perhaps someone could add those instructions there.

Reply:
This is being generated as part of the `injective-protobuf` package. See https://github.com/InjectiveLabs/cw-injective/blob/7a72491d76289172d7f4c61fc4876ef364c849c2/packages/injective-protobuf/build.rs#L20.

We'll soon publish this package to crates.io. That should solve your issues.

If there are other structs that you would need to be generated, feel free to open an issue / contribre a PR on that repo.

Reply:
Oh awesome thanks for the info. Currently this is the only one I need! Will submit an issue if anything else comes up. Thanks!

Reply:
Published. See https://crates.io/crates/injective-protobuf